assets/collage.css

```diff
@@ -29,11 +29,7 @@
 
 @media screen and (min-width: 750px) {
   .collage {
-    grid-template-columns: 2fr 1fr;
-  }
-
-  .collage--right {
-    grid-template-columns: 1fr 2fr;
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
This is to deal with the spacing difference with the different sections ? (multi columns, collages, etc)

Reply:
How I see how it fixes the spacing, nice 👌 

But it doesn't seem to work as expected when doing the `right large block` option